index.html

```diff
@@ -155,8 +256,8 @@ <h3 class="pcol">LET’S CONNECT</h3>
                             <img class="imageabout" src="csss.png" alt="Js"> <span class="lastdiv">HTML</span>
                         </div>
                     </li>
-                    <li><p class="aboutp">Frameworks <span class="greater">&gt;</span></p></li>
-                    <li><p class="aboutp">Skills <span class="greater">&gt;</span></p></li>
+                    <li class="lidesktop"><p class="aboutp">Frameworks <span class="greater">&gt;</span></p></li>
+                    <li class="lidesktop"><p class="aboutp">Skills <span class="greater">&gt;</span></p></li>
```

Comment:
- [ ] Please😃 try to add a suitable width for the left and right content here for example put the left content in 40% width and the right 60% to be like the Figma Design😊
- [ ] Also consider adding a suitable margin-top of the section about to be like the Figma design 💯

> Check below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/215075194-a9aa0e6b-63ee-4923-98e9-c3702857d26d.png) |   ![image](https://user-images.githubusercontent.com/83360636/215075247-5ebdaea4-df29-44be-8954-aeeadfe3257b.png)|

Reply:
suggested changes made to the project. Screen shot attached below for your kind reference. 
![image](https://user-images.githubusercontent.com/117091923/215103162-eb492c50-1239-43f1-bd60-16404977f7cf.png)